charts/nginx/templates/nginx-cp/nginx-cp-deployment.yaml

```
@@ -0,0 +1,139 @@
+####################################
+## NGINX Control Plane Deployment ##
+####################################
+{{- if .Values.global.controlplane.enabled }}
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: {{ template "nginx.fullname" . }}-cp
+  labels:
+    tier: {{ template "nginx.name" . }}
+    component: ingress-cp-controller
+    chart: {{ template "nginx.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    plane: controlplane
+spec:
+  replicas: {{ .Values.replicas }}
+  minReadySeconds: {{ .Values.minReadySeconds }}
+  selector:
+    matchLabels:
+      tier: {{ template "nginx.name" . }}
+      component: ingress-cp-controller
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        tier: {{ template "nginx.name" . }}
+        component: ingress-cp-controller
+        release: {{ .Release.Name }}
+        app: {{ template "nginx.name" . }}
+        version: {{ .Chart.Version }}
+        plane: controlplane
+      annotations:
+        traffic.sidecar.istio.io/includeInboundPorts: ""
+        checksum/configmap: {{ include (print $.Template.BasePath "/nginx-cp/nginx-cp-configmap.yaml") . | sha256sum }}
+        checksum/headers-configmap: {{ include (print $.Template.BasePath "/nginx-cp/nginx-cp-headers-configmap.yaml") . | sha256sum }}
+        {{- if .Values.global.istio.enabled }}
+        sidecar.istio.io/inject: "false"
+        {{- end }}
+{{- if .Values.global.podAnnotations }}
+{{ toYaml .Values.global.podAnnotations | indent 8 }}
+{{- end }}
+{{- if .Values.podAnnotations }}
+{{ toYaml .Values.podAnnotations | indent 8 }}
+{{- end }}
```

This is probably easier to understand:
```suggestion
        {{- if .Values.global.podAnnotations }}
        {{- toYaml .Values.global.podAnnotations | nindent 8 }}
        {{- end }}
        {{- if .Values.podAnnotations }}
        {{- toYaml .Values.podAnnotations | nindent 8 }}
        {{- end }}
```